AISKU/src/Initialization.ts

```diff
@@ -71,17 +71,15 @@ export class Initialization implements IApplicationInsights {
     /**
      * Log a user action or other occurrence.
      * @param {IEventTelemetry} event
-     * @param {{ [key:string]: any }} [customProperties]
      * @memberof Initialization
      */
-    public trackEvent(event: IEventTelemetry, customProperties?: { [key:string]: any }) {
-        this.appInsights.trackEvent(event, customProperties);
```

Comment:
customProperties wills stay Mark. It reflects part C.

Reply:
Added this part back in